lua/autorun/sh_vamp_convars.lua

```diff
@@ -0,0 +1,19 @@
+CreateConVar("ttt2_vamp_bloodtime", "60", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
+CreateConVar("ttt2_vamp_maxhealth", "250", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
+
+hook.Add("TTTUlxDynamicRCVars", "TTTUlxDynamicVampCVars", function(tbl)
```

Comment:
Please make sure that your identations are constant. They are all over the place in this hook.

Reply:
They seem constant (i used 1 space) Maybe I did not understand well you request(?)

Reply:
We use one tab. Therefore your indentation is completely different than the rest of the file

Reply:
E.g line 7 is a tab, line 8 are spaces

Reply:
Ok so I will make all Tab

Reply:
I changed all the spaces in the tab, see if it is correct now

Reply:
11 to 19 are still spaces

Reply:
anyway i think it's all fixed now